app/assets/stylesheets/application.scss

```diff
@@ -243,6 +244,10 @@ main {
   @include sr-only;
 }
 
+a[href^='http']:not([href*="fileyourstatetaxes.org" i]) {
```

Comment:
[DUST] CFA, is this a sufficient list of external links?

Reply:
In order to support GYR as well I'd like to parameterize this based on the request domain. I tested some steps to follow:
1. add a definition to `application_helper.rb`
```rb
  def request_domain
    request.domain
  end
```
2. update the logic: `a[href^='http']:not([href*=request_domain i])`

I think that should be sufficient to catch external links.